compass/landice/tests/thwaites/decomposition_test/__init__.py

```diff
@@ -0,0 +1,65 @@
+from compass.validate import compare_variables
+from compass.testcase import TestCase
+from compass.landice.tests.thwaites.run_model import RunModel
+
+
+class DecompositionTest(TestCase):
+    """
+    A test case for performing two MALI runs of the Thwaites setup,
+    with two different core counts.  The test case verifies that the
+    results of the two runs are identical.
+    """
+
+    def __init__(self, test_group):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.landice.tests.thwaites
+            The test group that this test case belongs to
+
+        """
+        name = 'decomposition_test'
+        subdir = '{}'.format(name)
+        super().__init__(test_group=test_group, name=name, subdir=subdir)
```

Comment:
This is fine and probably made sense copying the `greenland` test cases.  Since `subdir` defaults to being the same as `name`, you can just do:
```suggestion
        super().__init__(test_group=test_group, name='decomposition_test')
```
If you like the clarity of giving the `subdir` explicitly, I would at least take out the unnecessary string formatting:
```suggestion
        name = 'decomposition_test'
        subdir = name
        super().__init__(test_group=test_group, name=name, subdir=subdir)
```